superdesk/users/services.py

```diff
@@ -363,6 +363,19 @@ def get(self, req, lookup):
                 lookup['is_author'] = bool(int(is_author))
             else:
                 logger.warn('bad value of is_author argument ({value})'.format(value=is_author))
+
+        """filtering out inactive users and disabled users"""
+
+        params = json.loads(request.args.get('where')) if request and request.args.get('where') else {}
+
+        # Filtering inactive users
+        if not req.args.get('show_inactive') and params.get('is_active', True):
```

Comment:
not sure why you need the `params` here, wouldn't just `if not reg.args.get('show_inactive'):` do?

Reply:
> wouldn't just if not reg.args.get('show_inactive'): do?

No, the reason for this is if we want only the inactive_user then we will hit the URL as `users?where=is_active: false`, so now if we check only for the `if not reg.args.get('show_inactive'):` and not for params then this condition will get true and it will assign lookup['is_active'] as True and we will not get the result which we want.

Reply:
I think if we want inactive it should do `/users?where=is_active:false&show_inactive=true`, this `show_inactive` should be always used on pages like user management just `where` would be changing